packages/fxa-auth-server/test/local/routes/subscriptions/stripe.js

```diff
@@ -643,6 +649,45 @@ describe('DirectStripeRoutes', () => {
       }
     });
 
+    it('errors if the planCurrency does not match the paymentIntent currency', () => {
+      plan.currency = 'EUR';
+      directStripeRoutesInstance.stripeHelper.findPlanById.resolves(plan);
+
+      return directStripeRoutesInstance
+        .createSubscriptionWithPMI(VALID_REQUEST)
```

Comment:
Why not `await` on these? I thought most of the tests used `await` instead of promise chains.

Reply:
I saw other tests in the same block using the promise chains. I don't care either way. Changing it.